R/data.R

```diff
@@ -1,11 +1,12 @@
 #' Example dataset for lefser
 #'
 #' The ZellerG_2014 dataset contains microbiome count data for CRC patients and
-#' controls. It was for curatedMetagenomicData using the script in the package
-#' directory "data-raw".
+#' controls. It was for [curatedMetagenomicData][curatedMetagenomicData::curatedMetagenomicData]
+#' using the script in the package directory "data-raw".
```

Comment:
I would leave this as `curatedMetagenomicData` otherwise you'd have to move the package to `Imports` so that the link works (currently it is in `Suggests`). `lefser` can be installed without `curatedMetagenomicData` which would create a broken link.